msv/src/main/java/com/sun/msv/grammar/SimpleNameClass.java

```diff
@@ -55,16 +56,23 @@ public SimpleNameClass( StringPair name ) {
     }
     
     public SimpleNameClass( String namespaceURI, String localName ) {
+        this.namespaceURI    = namespaceURI; 
+        this.localName        = localName;
+        this.prefix = null;
```

Comment:
here prefix is set to null

Reply:
Here I will be consistently changing to ""! Thanks again!
I was a bit tempted to adopt toString() to return {namespaceURI}localName in case of a default namespace but do not want to change existing behavior of MSV.